### PR TITLE
Add tags to the tutorials section

### PR DESCRIPTION
**Pull request by CristhianMotoche**
### Trello Card
This Pull Request is related to this [Trello Card](https://trello.com/c/Q2lGOsPg/52-as-a-user-i-want-to-see-the-section-tags-link-in-the-blogspot-section-so-our-readers-have-option-to-read-more-articles-related)

### Screenshoots
- Tags in one blog post:
image
- List
image
- Route of tags
image

### Changes in the PR
- Tag generation under "tags/*.html"
- Add links for tags
- Add tags to blogs

@juanpaucar Could you take a look, please?

### CODE REVIEW

**Comment by CristhianMotoche** [comment]
[As a user, I want to see the section tags link in the Blogspot section so our readers have option to read more articles related](https://trello.com/c/Q2lGOsPg/52-as-a-user-i-want-to-see-the-section-tags-link-in-the-blogspot-section-so-our-readers-have-option-to-read-more-articles-related)

**Comment by CristhianMotoche** [comment]
@juanpaucar I think this is ready. Please, let me know if you want something to change. 👍

File: tutorials/haskell/bdd-in-haskell-with-hspec/tutorial.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: Behavior-driven development (BDD) in Haskell with Hspec
 published: 2016-06-14
 ghc: 7.10.3
 lts: 5.18
+tags: haskell, bdd
```
Comment (juanpaucar) [inline tutorials/haskell/bdd-in-haskell-with-hspec/tutorial.md]: It's probably a better idea to use `testing` or `test` instead of `bdd`

**Review by juanpaucar — COMMENTED** [review]
LGTM :shipit: Thanks

**Pull request merged**